blue-button.py/bluebutton/encoding.py

```diff
@@ -0,0 +1,129 @@
+"""
```

Comment:
There's no need to do this now - but we may want to consider having a separate encoding library within the LinuxForHealth org that's universally available or look into other 3rd party libraries.